index.js

```diff
@@ -155,31 +181,39 @@ async function underPressure (fastify, opts) {
 
   function onRequest (req, reply, next) {
     if (checkMaxEventLoopDelay && eventLoopDelay > maxEventLoopDelay) {
-      sendError(reply, next)
-      return
+      req[isUnderPressureError] = true
     }
 
     if (checkMaxHeapUsedBytes && heapUsed > maxHeapUsedBytes) {
-      sendError(reply, next)
-      return
+      req[isUnderPressureError] = true
     }
 
     if (checkMaxRssBytes && rssBytes > maxRssBytes) {
-      sendError(reply, next)
-      return
+      req[isUnderPressureError] = true
     }
 
     if (!externalsHealthy) {
-      sendError(reply, next)
-      return
+      req[isUnderPressureError] = true
     }
 
     if (checkMaxEventLoopUtilization && eventLoopUtilized > maxEventLoopUtilization) {
-      sendError(reply, next)
-      return
+      req[isUnderPressureError] = true
+    }
+
+    if (
+      [
+        opts.exposeStatusRoute.url,
+        opts.exposeStatusDetailsRoute.url
+      ].includes(req.url)
+    ) {
```

Comment:
This should be a little bit performant:

```suggestion
    if (opts.exposeStatusRoute.url === req.url ||  opts.exposeStatusDetailsRoute.url === req.url) {
```

I would use `includes` when the `if` would be unreadable